code/datums/black_market_sellable.dm

```diff
@@ -69,78 +71,361 @@ var/list/black_market_sellables = list()
+	demand_min = 1
+	demand_max = 1
+	price_min = 700
+	price_max = 800
+	sps_chance = 0
+	display_chance = 100
+	teleport_modifier = 3
+
+/datum/black_market_sellable/animals/ian/purchase_check(var/obj/input, var/mob/user)
+	if(istype(input,/mob/))
+		var/mob/ian = input
+		if(ian.isDead())
+			return "Ian is dead. This just won't do."
+	return VALID
 	
-/datum/black_market_sellable/weapons
-	category = "Firearms and Weaponry"
+/datum/black_market_sellable/animals/ian/after_sell(var/obj/input, var/mob/user)
+	spawn(rand(350,650))
+		command_alert(new /datum/command_alert/ian_sold(user))
+		
+/datum/black_market_sellable/animals/sasha
+	name = "Sasha the Doberman"
+	item = /mob/living/simple_animal/corgi/sasha
+	no_children = 1
+	demand_min = 1
+	demand_max = 1
+	price_min = 300
+	price_max = 400
+	sps_chance = 0
+	display_chance = 35
+	teleport_modifier = 3
 	
-/datum/black_market_sellable/weapons/ion_rifle
-	name = "Ion Rifle"
-	item = /obj/item/weapon/gun/energy/ionrifle
+/datum/black_market_sellable/animals/punpun
+	name = "Punpun the Monkey"
+	item = /mob/living/carbon/monkey/punpun
 	no_children = 1
 	demand_min = 1
-	demand_max = 3
+	demand_max = 1	
+	price_min = 50
+	price_max = 100
+	sps_chance = 5
+	display_chance = 35
+	teleport_modifier = 2
+
+/datum/black_market_sellable/animals/poly
+	name = "Poly the Parrot"
+	item = /mob/living/simple_animal/parrot/Poly
+	no_children = 1
+	demand_min = 1
+	demand_max = 1	
+	price_min = 150
+	price_max = 300
+	sps_chance = 15
+	display_chance = 35
+	teleport_modifier = 2
+		
+/datum/black_market_sellable/animals/corpus
+	name = "Corpus the Snake"
+	item = /mob/living/simple_animal/cat/snek/corpus
+	no_children = 1
+	desc = "sssSSSSsss"
+	demand_min = 1
+	demand_max = 1	
 	price_min = 100
 	price_max = 200
-	display_chance = 100
+	sps_chance = 15
+	display_chance = 35
+	teleport_modifier = 2
+	
+/datum/black_market_sellable/animals/runtime
+	name = "Runtime the Cat"
+	item = /mob/living/simple_animal/cat/Runtime
+	no_children = 1
+	demand_min = 1
+	demand_max = 1
+	price_min = 150
+	price_max = 200
+	sps_chance = 15
+	display_chance = 35
+	teleport_modifier = 2	
 
-/datum/black_market_sellable/weapons/energy_gun
-	name = "Energy Gun"
-	item = /obj/item/weapon/gun/energy/
-	no_children = 0
-	desc = "Paying double for fully charged weapons, but guns with no charge will be denied."
+/datum/black_market_sellable/animals/salem
+	name = "Salem the Neglected Cat"
+	item = /mob/living/simple_animal/cat/salem
+	no_children = 1
 	demand_min = 1
-	demand_max = 3
+	demand_max = 1
 	price_min = 100
+	price_max = 150
+	sps_chance = 15
+	display_chance = 35
+	teleport_modifier = 2		
+
+	
+	
+	
+/datum/black_market_sellable/misc
+	category = "Miscellaneous Items"
+	
+/datum/black_market_sellable/misc/tome
+	name = "Occult Tome"
+	item = /obj/item/weapon/tome
+	no_children = 1
+	desc = "Looking for the one from the cult of Nar-Sie. Teleporting magical items will likely send an alert to Centcomm."
```

Comment:
`magical` is a really shitty term, try something like `occultic`.

Reply:
Changed to be more professional.